agent/engine/docker_task_engine.go

```diff
@@ -1658,6 +1666,13 @@ func (engine *DockerTaskEngine) provisionContainerResources(task *apitask.Task,
 		}
 	}
 
+	if task.IsServiceConnectEnabled() && task.IsNetworkModeBridge() && container == task.GetServiceConnectContainer() {
+		// TODO: CNI integration for SC bridge-mode task
```

Comment:
nit: can we do `// TODO [SC]:...` so that it's easier to identify SC TODOs later?